source/fundamentals/crud/read-operations/geo.txt

```diff
@@ -0,0 +1,310 @@
+===================
+Search Geospatially
+===================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+Overview
+--------
+
+On this page you will learn how to search **geospatial data** with the
+MongoDB Java Driver, and the different geospatial data formats supported by MongoDB.
+
+Geospatial data is data that represents a geographical location on
+the surface of the Earth. Examples of geospatial data include:
+
+- Locations of movie theaters
+- Borders of countries
+- Routes of bicycle rides
+- Dog exercise areas in New York City
+
+Coordinates on Earth
+--------------------
+
+To store and query your geospatial data in MongoDB, use **GeoJSON**. GeoJSON is
+a data format created by the Internet Engineering Task Force (IETF). 
+
+Here is the location of MongoDB headquarters in GeoJSON:
+
+.. code-block:: json
+
+   "MongoDB Headquarters" : {
+      "type": "point",
+      "coordinates": [-73.986805, 40.7620853]
+   }
+
+For definitive information on GeoJSON, see the
+:rfc:`official IETF specification <7946>`.
+
+.. external resource
+
+GeoJSON Positions
+~~~~~~~~~~~~~~~~~
+
+A position represents a single place on Earth, and exists in code as an array
+containing two or three number values: 
+
+- Longitude in the first position (required)
+- Latitude in the second position (required)
+- Elevation in the third position (optional)
+
+.. tip:: Longitude then Latitude
+
+  GeoJSON orders coordinates as longitude first and latitude second. This may
+  be surprising as geographic coordinate system conventions generally list
+  latitude first and longitude second. Make sure to check what format any other
+  tools you are working with use. Popular tools such as OpenStreetMap and Google
+  Maps list coordinates as latitude first and longitude second. 
+  
+  Be careful when moving geospatial data between GeoJSON and other
+  formats, as you may need to swap your longitude and latitude values. 
+
+GeoJSON Types
+~~~~~~~~~~~~~
+
+Your GeoJSON object's type determines its geometric shape. Geometric shapes are
+made up of positions.
+
+Here are some common GeoJSON types and their corresponding positions:
```

Comment:
Thanks for the note. I am not 100% sure I am understanding your comment, but I agree that "corresponding positions" is a poor choice of words.

Fix: https://github.com/mongodb/docs-java/pull/78/commits/4dc09e9f8a33b7a80a77c691e7556e387df689fa

> It could be confused with geo positions, and readers may not directly relate a position to a linestring or poly, for example

Not sure what specifically the means, are you saying that readers may have trouble seeing that GeoJSON positions can form more descriptive geometric structures? If this is a correct reading, does my edit provide a satisfactory fix?